crates/weechat-sys/build.rs

```diff
@@ -25,7 +25,7 @@ fn build(file: &str) -> Result<Bindings, BindgenError> {
         "WEECHAT_HOOK_SIGNAL_INT",
         "WEECHAT_HOOK_SIGNAL_POINTER",
     ];
-    let mut builder = bindgen::Builder::default().rustfmt_bindings(true);
+    let mut builder = bindgen::Builder::default();
```

Comment:
What's the rationale for removing the formatting?

Reply:
This option is deprecated and `Builder::formatter` is preferred, but the default formatter is `Rustfmt` anyway
https://docs.rs/bindgen/0.70.1/bindgen/struct.Builder.html#method.rustfmt_bindings

Reply:
(I should've noted that in commit message, my fault)

Reply:
Ah indeed, thanks for letting me know.